src/features/CourtCaseDetails/Sidebar/Exceptions.tsx

```diff
@@ -0,0 +1,72 @@
+import { GridCol, GridRow } from "govuk-react"
+import { createUseStyles } from "react-jss"
+import getExceptionInfo from "utils/getExceptionInfo"
+import ActionLink from "components/ActionLink"
+import type { AnnotatedHearingOutcome } from "@moj-bichard7-developers/bichard7-next-core/build/src/types/AnnotatedHearingOutcome"
+
+interface Props {
+  aho: AnnotatedHearingOutcome
+}
+
+const useStyles = createUseStyles({
+  exceptionRow: {
+    "& .exception-details-column": {
+      "& .exception-field": {
+        display: "block",
+        fontSize: "19px",
+        fontWeight: "bold"
+      },
+      "& .exception-details": {
+        fontSize: "16px"
+      }
+    }
+  }
+})
+
+const getExceptionTitle = (path: (string | number)[]) => {
+  const offenceIndex = path.findIndex((p) => p === "Offence")
+  let location = ""
+  if (offenceIndex > 0) {
+    const offenceOrderIndex = Number(path[offenceIndex + 1]) + 1
+    location = `Offence ${offenceOrderIndex}`
+  } else if (path.includes("Case information")) {
+    location = "Case"
+  } else if (path.includes("Hearing")) {
+    location = "Hearing"
+  }
+
+  const fieldName = String(path[path.length - 1])
+  const fieldNameWords = fieldName.match(/([A-Z]+[a-z]+)/g)
+  const formattedFieldName = fieldNameWords
+    ? fieldNameWords[0] + fieldNameWords.join(" ").slice(fieldNameWords[0].length).toLowerCase()
+    : fieldName
+
+  return `${formattedFieldName} (${location})`
+}
+
+const Exceptions: React.FC<Props> = ({ aho }) => {
```

Comment:
We can write it the other way (I've added a comment above) & avoid using `React.FC`.